### ISSUE TITLE
Should we refer to the AMQP driver as "AMQP" or "RabbitMQ"?

### ISSUE THREAD

**Issue by andreasjansson** [replicate/cog#473]
While AMQP is an open standard, the most widely used implementation is RabbitMQ.

RabbitMQ supports [AMQP version 0.9.1](https://www.cloudamqp.com/blog/what-is-amqp-and-why-is-it-used-in-rabbitmq.html) and it's unlikely that RabbitMQ will ever use AMQP 1.0 (though it can support 1.0 via a plugin). Version 1.0 is a major breaking change from 0.9.1.

There are [other implementations](https://en.wikipedia.org/wiki/Advanced_Message_Queuing_Protocol#Implementations) of AMQP, some of which support 1.0 and some who don't.

This means that the name "AMQP" is inherently ambiguous, and we would have to either refer to it as "AMQP 0.9.1" or "AMQP 1.0" in our documentation and code. The alternative is that we simply call it "the RabbitMQ driver".

"RabbitMQ" is unambiguous and more well-known than "AMQP". It's probably worth sacrificing the generality we'd get from "AMQP", given that non-RabbitMQ implementations are rarely used.

**Comment by zeke** [comment]
Single anecdotal data point: I'm familiar with RabbitMQ but had never heard of AMQP before we started discussing it.

**Comment by evilstreak** [comment]
For now we're going to stick with Redis as our queueing system. I'm closing this issue because it's no longer relevant.

Some of the discussion in #443 outlines an approach where we only expose an HTTP API from Cog, and make the queueing system an internal implementation detail. That seems like a good direction to go in, so we're going to make changes to the queueing API we currently use to match the HTTP API, but we aren't going to swap from Redis to AMQP.
